### PR TITLE
Feature/enumerable

### PR DESCRIPTION
**Pull request by topeogunleye**
### Added Changes
- Added Enumerable Module
- Added MyList class

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  include Enumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  #  method #each that yields successive members of @list
+  def each(&block)
+    @list.each(&block)
+  end
+end
```
Comment (Dipeshtwis) [inline my_list.rb]: - [ ] Inside `each` method you have to implement your logic to to behave like each. kindly implement this.

Like this:

```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+
+> 
+
+
+## Built With
+
+- Ruby
+
+## Authors
+
```
Comment (Dipeshtwis) [inline README.md]: - [ ] I would recommend including the steps to clone and run the project locally to make the README professional.

```
git clone https://github.com/cwaku/enumerables.git
cd enumerables
ruby my_list.rb
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @topeogunleye & @cwaku, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Temi & Ricky, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**